tests/integration_tests/with_db/test_audit_table.py

```diff
@@ -0,0 +1,70 @@
+from dataclasses import dataclass
+
+import pyexasol
+import pytest
+from exasol.python_extension_common.deployment.temp_schema import temp_schema
+
+from exasol.analytics.audit.audit import AuditTable
+from exasol.analytics.audit.columns import BaseAuditColumns
+from exasol.analytics.schema import (
+    SchemaName,
+    Table,
+    TableNameImpl,
+    decimal_column,
+    varchar_column,
+)
+
+
+@pytest.fixture(scope="session")
+def db_schema(pyexasol_connection):
+    with temp_schema(pyexasol_connection) as db_schema:
+        yield db_schema
+
+
+@dataclass
+class ExaAllColumns:
+    connection: pyexasol.ExaConnection
+    schema: str
+
+    def query(self, table_name: str) -> dict[str, str]:
+        raw = self.connection.execute(
+            "SELECT COLUMN_NAME, COLUMN_TYPE FROM EXA_ALL_COLUMNS"
+            f" WHERE COLUMN_SCHEMA='{self.schema}'"
+            f" AND COLUMN_TABLE='{table_name}'"
+        ).fetchall()
+        return {c[0]: c[1] for c in raw}
+
+
+@pytest.fixture
+def exa_all_columns(pyexasol_connection, db_schema):
+    return ExaAllColumns(pyexasol_connection, db_schema)
+
+
+def test_create_table(pyexasol_connection, db_schema, exa_all_columns):
```

Comment:
this probably should be tested in its own file

Reply:
See next push.